tests/automation/message_checks.spec.ts

```diff
@@ -18,246 +21,266 @@ import {
+    await clickOnElement({
+      window: aliceWindow1,
+      strategy: 'data-testid',
+      selector: 'send-message-button',
+    });
+    await sleepFor(1000);
+    await clickOnMatchingText(bobWindow1, 'Click to download media');
+    await clickOnTestIdWithText(bobWindow1, 'session-confirm-ok-button');
+  },
+);
 
-  await windowA.setInputFiles(
-    "input[type='file']",
-    'tests/automation/fixtures/test-gif.gif',
-  );
-  await sleepFor(100);
-  await clickOnElement({
-    window: windowA,
-    strategy: 'data-testid',
-    selector: 'send-message-button',
-  });
-  await sleepFor(1000);
-  await clickOnMatchingText(windowB, 'Click to download media');
-});
+test_Alice_1W_Bob_1W(
+  'Send GIF 1:1',
+  async ({ alice, aliceWindow1, bob, bobWindow1 }) => {
+    // const testReply = `${bob.userName} to ${alice.userName}`;
+    await createContact(aliceWindow1, bobWindow1, alice, bob);
 
-sessionTestTwoWindows('Send long text 1:1', async ([windowA, windowB]) => {
-  const [userA, userB] = await Promise.all([
-    newUser(windowA, 'Alice'),
-    newUser(windowB, 'Bob'),
-  ]);
+    await aliceWindow1.setInputFiles(
+      "input[type='file']",
+      'tests/automation/fixtures/test-gif.gif',
+    );
+    await sleepFor(100);
+    await clickOnElement({
+      window: aliceWindow1,
+      strategy: 'data-testid',
+      selector: 'send-message-button',
+    });
+    await sleepFor(1000);
+    await clickOnMatchingText(bobWindow1, 'Click to download media');
+  },
+);
 
-  const testReply = `${userB.userName} replying to long text message from ${userA.userName}`;
-  const longText =
-    // eslint-disable-next-line max-len
-    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum quis lacinia mi. Praesent fermentum vehicula rhoncus. Aliquam ac purus lobortis, convallis nisi quis, pulvinar elit. Nam commodo eros in molestie lobortis. Donec at mattis est. In tempor ex nec velit mattis, vitae feugiat augue maximus. Nullam risus libero, bibendum et enim et, viverra viverra est. Suspendisse potenti. Sed ut nibh in sem rhoncus suscipit. Etiam tristique leo sit amet ullamcorper dictum. Suspendisse sollicitudin, lectus et suscipit eleifend, libero dui ultricies neque, non elementum nulla orci bibendum lorem. Suspendisse potenti. Aenean a tellus imperdiet, iaculis metus quis, pretium diam. Nunc varius vitae enim vestibulum interdum. In hac habitasse platea dictumst. Donec auctor sem quis eleifend fermentum. Vestibulum neque nulla, maximus non arcu gravida, condimentum euismod turpis. Cras ac mattis orci. Quisque ac enim pharetra felis sodales eleifend. Aliquam erat volutpat. Donec sit amet mollis nibh, eget feugiat ipsum. Integer vestibulum purus ac suscipit egestas. Duis vitae aliquet ligula.';
+test_Alice_1W_Bob_1W(
+  'Send long text 1:1',
+  async ({ alice, aliceWindow1, bob, bobWindow1 }) => {
+    const testReply = `${bob.userName} replying to long text message from ${alice.userName}`;
+    const longText =
```

Comment:
Could we have a constants with dummy text data that you can import? Keeping the tests clean

Reply:
I guess a comment for @burtonemily

Reply:
Guess we can resolve for now and Emily can have a look at this later.